assets/javascripts/kitten/components/modals/next/index.js

```diff
@@ -0,0 +1,307 @@
+  
+    @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+      margin: ${pxToRem(100)} auto;
+      padding: ${pxToRem(80)} ${oneGridCol};
+      width: calc((100vw - ${pxToRem(paddingPlusGutters)}) + (${pxToRem(
+  GUTTER,
+)} * 11))
+    }
+  
+    @media (min-width: ${pxToRem(ScreenConfig.L.min)}) {
+      margin: ${pxToRem(100)} auto;
+      padding: ${pxToRem(80)} ${oneGridCol};
+      ${props => css`
+        width: calc(
+          ((100vw - ${pxToRem(paddingPlusGutters)}) / 12 + ${pxToRem(GUTTER)}) *
+            ${props.cols} - ${pxToRem(GUTTER)}
+        );
+      `}
+    }
+    
+    @media (min-width: ${pxToRem(ScreenConfig.XL.min)}) {
+    ${props => css`
+      width: ${pxToRem(
+        ((CONTAINER_MAX_WIDTH - paddingPlusGutters) / 12 + GUTTER) *
+          props.cols -
+          GUTTER,
+      )};
+    `}
+    }
+  }
+
+  .k-Modal__close {
+    position: absolute;
+    top: 0;
+    right: ${pxToRem(40)};
+    @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+     right: ${pxToRem(50)};
+  }
+
+    button {
+      margin: 0;
+    }
+  }
+  
+  .k-Modal-content {
+    transform: scale(0.94);
+  }
+
+  .k-Modal__overlay {
+    position: fixed;
+    z-index: 110;
+    overflow: scroll;
+    display: block;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    opacity: 0;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(34, 34, 34, .8);
+  }
+  
+  .k-Modal__overlay--afterOpen {
+    transition: opacity .3s ease;
+    opacity: 1;
+  }
+  .k-Modal--afterOpen {
+    transition: opacity .3s ease, transform .3s ease;
+    transform: scale(1);
+    opacity: 1;
+  }
+
+  .k-Modal__overlay--beforeClose {
+    opacity: 0;
+  }
+  .k-Modal--beforeClose {
+    transition: opacity .3s ease, transform .5s ease;
+    transform: scale(1.06);
+    opacity: 0;
+  }
+`
+
+const ModalTitle = ({ children }) => (
+  <Title
+    modifier="quaternary"
+    margin={false}
+    tag="p"
+    className="k-u-margin-bottom-singleHalf--important k-u-align-center"
+  >
+    {children}
+  </Title>
+)
+
+const ModalParagraph = ({ children, align }) => (
+  <StyledParagraph
+    modifier="quaternary"
+    margin={false}
+    style={{ textAlign: align }}
+    tag="p"
+    className="k-u-margin-bottom-triple k-u-margin-bottom-quadruple@s-up"
+  >
+    {children}
+  </StyledParagraph>
+)
+
+ModalParagraph.propTypes = {
+  align: PropTypes.oneOf(['center', 'left', 'right', 'justify']),
+}
+
+ModalParagraph.defaultProps = {
+  align: 'center',
+}
+
+const Actions = styled.div`
+  display: flex;
+  flex-direction: column;
+  @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+    flex-direction: row;
+  }
+`
+
+const ModalButton = styled(props => <Button big fluid {...props} />)`
+  :not(:last-child) {
+    margin-bottom: ${pxToRem(20)};
+  }
+  @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+    :not(:last-child) {
+      margin-bottom: 0;
+    }
+    &:not(:first-child) {
```

Comment:
```suggestion
    :not(:first-child) {
```

pas primordial mais ça fait unité avec les précédents styles :)